starboard/nplb/player_get_audio_configuration_test.cc

```diff
@@ -0,0 +1,198 @@
+// Copyright 2023 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <vector>
+
+#include "starboard/nplb/player_test_fixture.h"
+#include "starboard/string.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+#if SB_API_VERSION >= SB_MEDIA_ENHANCED_AUDIO_API_VERSION
+
+bool operator==(const SbMediaAudioConfiguration& left,
+                const SbMediaAudioConfiguration& right) {
+  return memcmp(&left, &right, sizeof(SbMediaAudioConfiguration)) == 0;
+}
+
+namespace starboard {
+namespace nplb {
+namespace {
+
+using ::testing::ValuesIn;
+
+typedef SbPlayerTestFixture::GroupedSamples GroupedSamples;
+
+#if SB_API_VERSION >= SB_MEDIA_ENHANCED_AUDIO_API_VERSION
+
+class SbPlayerGetAudioConfigurationTest
+    : public ::testing::TestWithParam<SbPlayerTestConfig> {
+ public:
+  void ReadAudioConfigurations(
+      const SbPlayerTestFixture& player_fixture,
+      std::vector<SbMediaAudioConfiguration>* configurations) {
```

Comment:
static std::vector<SbMediaAudioConfiguration> GetAudioConfigurations(
      const SbPlayerTestFixture& player_fixture)

Reply:
It's to use ASSERT_LT in the function.

Reply:
Pending removal of ASSERT_LT per the above discussion.

Reply:
In this case let's add `SB_CHECK(configurations->empty());` or `configurations->clear()` here.